frontend/src/components/utils/NavBar/index.tsx

```diff
@@ -193,23 +193,37 @@ const NavBar = ({ headersData }: Props): ReactElement => {
       </Grid>
     </Grid>
   );
-
```

Comment:
Can you help revert those formatting changes (same with other)? Keeping line in middle enhances the readability of our code base.